bundler/tasks.py

```diff
@@ -62,6 +62,48 @@ def gen_reference(output_dir, dataset, phase_id):
     pass
 
 
+def load_metadata_files(data, tmp_dir, name):
+    for ext, ext_name in [(data.input, 'feat'), (data.target, 'label')]:
+        for axis, axis_name in [(ext.cols, 'cols'), (ext.rows, 'rows')]:
+            if axis is not None:
+                for content, cont_name in [(axis.types, 'type'),
+                                           (axis.names, 'name'),
+                                           (axis.doc, 'info')]:
+                    if content is not None:
+                        filename = '%s_%s.%s' % (name, ext_name, cont_name)
+
+                        if cont_name == 'info':
+                            filename = '%s_public.info' % name
+
+                        content.raw_content.open()
+                        copy_file_field(content.raw_content,
+                                        path.join(tmp_dir, filename))
+                        content.raw_content.close()
+
+
+def create_info_file(challenge, tmp_dir):
```

Comment:
I have no choice for that ^^
The dataset can contain a .info file, which is defined with this "format".